guide/popular-topics/permissions.md

```diff
@@ -324,6 +324,34 @@ If you want to know how to work with the returned Permissions objects keep readi
 The <branch version="11.x" inline>[Permissions](https://discord.js.org/#/docs/main/v11/class/Permissions)</branch><branch version="12.x" inline>[Permissions](https://discord.js.org/#/docs/main/stable/class/Permissions)</branch> object is a discord.js class containing a permissions bit field and a bunch of utility methods to manipulate it easily.
 Remember that using these methods will not manipulate permissions, but create a new instance representing the changed bit field.
 
+### Displaying permissions as an array
+
+Discord.js provides a `toArray()` method, which can be used to convert a `Permissions` object into an array containing permission flags. This is useful if you want to display/list them and it enables you to use other array methods. For example:
+
+```js
+const memberPermissions = member.permissions.toArray();
+const rolePermissions = role.permissions.toArray();
+//	Will return something along the lines of ['SEND_MESSAGES', 'ADD_REACTIONS', 'CHANGE_NICKNAME'] ...etc...
+```
+
+The meaning of the return value of `toArray()` can vary, depending on where the `Permissions` object is coming from. In the examples above, we grabbed a `GuildMember`'s and a `Role`'s [base](https://discordjs.guide/popular-topics/permissions.html#base-permissions) permissions, which is basically what a member/role can do, by default. However, you can also view things such as denied/allowed channel overwrites, and the meaning of the resulting array is different.
```

Comment:
- The meaning does not vary, #toArray always represent the permissions present in the permission flags present in the underlying bit field of the Permissions instance it's called on. Maybe you mean the right thing, but it's worded very confusingly.
- wording: "grabbed" does not read well
- wording: "The base permissions of a GuildMember and Role"
- you can omit the last part, it's worded rather confusingly and you link to the explanation in the terminology primer